plugins/inputs/win_perf_counters/README.md

```diff
@@ -19,7 +19,19 @@ as counters used when performance monitoring
  There are a lot of other good objects to monitor, if you know what to look for.
  This file is likely to be updated in the future with more examples for
  useful configurations for separate scenarios.
-
+ 
+ ### Schema
+ 
+ *Measurement name* is specified per performance object or `win_perf_counters` by default.
+ 
+ *Tags:*
+  - computer - computer name, if specified in the `Computers` parameter, otherwise omitted
+  - objectname - normalized name of the performance object
+  - instance - instance name, if performance object supports multiple instances, otherwise omitted
```

Comment:
What is the drawback/limitation to always setting `instance`?

Reply:
Instance part of performance object that has only single instance is empty.